src/main/java/ml/shifu/shifu/container/obj/EvalConfig.java

```diff
@@ -242,6 +243,22 @@ public void setGbtConvertToProb(Boolean gbtConvertToProb) {
         this.gbtConvertToProb = gbtConvertToProb;
     }
 
+    /**
+     * @param scoreScale
+     *            the scoreScale to set
+     */
+    public void setScoreScale(Long scoreScale) {
+        this.scoreScale = scoreScale;
```

Comment:
is it double type? and if you new one, is it in final eval json file part of ModelConfig.json?